lib/chai/utils/addProperty.js

```diff
@@ -40,8 +39,9 @@ module.exports = function (ctx, name, getter) {
 
   Object.defineProperty(ctx, name,
     { get: function addProperty() {
+        var keep_ssfi = flag(this, 'keep_ssfi');
```

Comment:
Shouldn't we test if `_super` is really in the stack trace when we've got properties/methods overwritten?
What do you guys think about this?

Tests are okay, now, but maybe we could have this too.